main/ertp/ERTP-Guide/Overview.md

```diff
@@ -0,0 +1,276 @@
+transferred exactly the same way and with exactly the same security properties. 
+
+ERTP uses
+*[object capabilities](https://agoric.com/documentation/glossary/#object-capabilities)*
+to enforce access control. If your program has a reference to an
+object, it can call methods on that object. If it doesn't have a
+reference, it can't. For more on object capabilities, see
+[Chip Morningstar's post](http://habitatchronicles.com/2017/05/what-are-capabilities/).
+
+## Objects Overview
+
+![ERTP object relationships](relationships.svg)  
+
+There are eight fundamental ERTP objects, two of which are parts of
+another, and one which is used by that same object. For each entry,
+the name is linked to its section in this document. The following "&"
+symbol is linked to the term's [Glossary]() entry.
+
+- **[Amount](#amount)** **[&](https://agoric.com/documentation/glossary/#amount)**:
+  the representation of an asset, made up of a
+  *[value](#value)* *[&](https://agoric.com/documentation/glossary/#value)*  (the
+  size of the amount) and a
+  *[brand](#brand)* *[&](https://agoric.com/documentation/glossary/#brand)* (the
+  kind of asset). You can think of these as the "how many" and the
+  "what" of an asset.
+- **[AmountMath](#amountmath)** **[&]((https://agoric.com/documentation/glossary/#amountmath)**:
+Methods to do math operations on `amount` objects.
+- **[Purse](#purse)** **[&](https://agoric.com/documentation/glossary/#purse)**: an
+  object for long term `amount` object storage. 
+- **[Payment](#payment)** **[&](https://agoric.com/documentation/glossary/#payment)**:
+  an object for transfering an `amount` object to another party.
+- **[Mint](#mint)** **[&](https://agoric.com/documentation/glossary/#mint)**: an
+  object that creates new `amount` objects of a specific brand. Each brand has
+  a one to one relationship with a mint and vice versa.
+- **[Issuer](#issuer)** **[&](https://agoric.com/documentation/glossary/#issuer)**: 
+an object linked to a single `mint` object that creates empty `purse` and
+`payment` objects for that mint's brand and operates on payments of
+that brand. 
+
+## Fungible and Non-Fungible Assets
+
+There are two kinds of assets,
+*[fungible](https://agoric.com/documentation/glossary/#fungible)* and
+*[non-fungible](https://agoric.com/documentation/glossary/#non-fungible)*. 
+
+Fungible assets are interchangeable. For example, if you have 100
+one-dollar bills and need to pay someone 5 dollars, it doesn't matter
+which five of your one-dollar bills you give them. 
+
+Non-fungible assets are of the same type, but are not interchangeable
+and specific items must be used. For example, you might have 100
+tickets. But someone wanting to buy a ticket from you will want one
+for a specific date, a specific theater, a specific time, and let's
+say a specific seat (these aren't General Admission tickets). 
+
+You can't just pull out a ticket at random and sell it to them, since
```

Comment:
I think the reader won't agree with this as written. I think we mean that tickets between shows are not fungible, but tickets within shows might be for the user. We should explain this in a different way so it's more easily agreed with.